bip-0374.mediawiki

```diff
@@ -118,6 +118,12 @@ This proposal is compatible with all older clients.
 A reference python implementation is included [https://github.com/bitcoin/bips/blob/master/bip-0374/reference.py here].
 Test vectors can be generated by running <code>./bip-0374/gen_test_vectors.py</code> which will produce a CSV file of random test vectors for both generating and verifying proofs. These can be run against the reference implementation with <code>./bip-0374/run_test_vectors.py</code>.
 
+== Changelog ==
+
+To help implementors understand updates to this BIP, we keep a list of substantial changes.
```

Comment:
s/implementors/implementers/ (see https://github.com/bitcoin/bips/pull/1766)

(or simply drop this line)